src/Cache.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\Gallerya\Cache.
+ */
+
+namespace Netzstrategen\Gallerya;
+
+/**
+ * Adds transient cache.
+ */
+class Cache {
+
+  /**
+   * The slider variation transient key prefix.
+   *
+   * @var string
+   */
+  const VARIATION_SLIDER_CACHE_KEY_PREFIX = Plugin::PREFIX . '_variation_attachments_for_';
+
+  /**
+   * Queries for cached markup, if none found, creates it for next time.
+   */
+  public static function get_cached_variation_slider_markup($product) {
```

Comment:
In general, we use camel-case for methods names, if I am not wrong. We just keep snake-case if we use a filter/action (WordPress hook) name as the method name.